src/translations/translationMetadata.json

```diff
@@ -91,7 +91,7 @@
     "nativeName": "Norsk Nynorsk"
   },
   "no": {
-    "nativeName": "Norsk"
+    "nativeName": "Norsk Bokmål"
```

Comment:
This is not correct. According to ISO 639, `no` stands for Norsk. Norsk Bokmål would be country code `nb`.

https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes

Reply:
So technically, the correct thing to do would be to add this change and also change the country code to be `nb` instead of `no`. But that would require an update of all Norwegian keys in our translation system 😱

Reply:
Yes, that would be the better fix. But it has wider reaching effects. Also might invalidate the users' already set-language if not "fixing" that on upgrade or some-such.

This is actually a very common problem in a lot of free software (probably proprietary as well) - since projects use `no` for Bokmål at the start which is technically wrong. :man_shrugging: